book/src/guest-libs/k256.md

```diff
@@ -0,0 +1,56 @@
+# K256
+
+The K256 guest library uses [`openvm-ecc-guest`](../custom-extensions/ecc.md) to provide elliptic curve operations over the Secp256k1 curve.  
```

Comment:
mention this is meant to be a patch of the `k256` crate and can be swapped in for signature verification usage. Signing from private key is not supported